client/src/Components/AboutMe/AboutMe.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import "./AboutMe.css";
+
+{
+  /* Two Flex box containers, about half n half width of page*/
+}
+
+function AboutMe() {
+  return (
+    <div id="AboutPage">
+      <div className="profileImageLeft">
+        <img src="../Images/IMG_1025.jpg" className="headShot"></img>
+      </div>
+      <div className="AboutParagraph">
+        <h2 id="About">About Me</h2>
+        <br></br>
```

Comment:
Use `justify-content` or `margin` for things like this

using `<br>` is clunky and not meant for this type of use